anomalydetection/features.py

```diff
@@ -20,3 +20,12 @@ def lag_time_series(time_series: pd.Series, lags):
         lagged_time_series[str(lag)] = time_series.shift(lag)
 
     return pd.concat(lagged_time_series, axis=1)
+
+
+def create_dataset(X, y, time_steps=1):
+    Xs, ys = [], []
+    for i in range(len(X) - time_steps):
```

Comment:
I'm not sure what the difference is between this and using the shift operator in Pandas?

Reply:
Don't think there is any. Maybe speed? I just copied from a course notebook.